simulation.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+import replay_log
+import simulation_util
+
+SOUCE_IP_INDEX =  2
+DESTINATION_IP_INDEX = 3
+
+def run_simulation(request_log_file):
+  # replaying request log
+  start_time, end_time = replay_log.simulate_requests(request_log_file)
+  # collect server logs
+  logs = get_server_logs(start_time, end_time)
+  # calculate the average latency
+  latency_sum = 0
+  request_count = 0
+  for log in logs:
+    client_ip = log[SOURCE_INDEX]
+    server_ip = log[DESTINATION_INDEX]
+    distance = get_distance(client_ip, server_ip)
+    unit = 1000.0
+    latency = distance / unit
```

Comment:
yes. Latency is actually in 1000 km unit. Otherwise latency can be as large as thousands, which looks wired...


> On Mar 16, 2015, at 12:51, Chris Baik <notifications@github.com> wrote:
> 
> In simulation.py:
> 
> > +DESTINATION_IP_INDEX = 3
> > +
> > +def run_simulation(request_log_file):
> > +  # replaying request log
> > +  start_time, end_time = replay_log.simulate_requests(request_log_file)
> > +  # collect server logs
> > +  logs = get_server_logs(start_time, end_time)
> > +  # calculate the average latency
> > +  latency_sum = 0
> > +  request_count = 0
> > +  for log in logs:
> > +    client_ip = log[SOURCE_INDEX]
> > +    server_ip = log[DESTINATION_INDEX]
> > +    distance = get_distance(client_ip, server_ip)
> > +    unit = 1000.0
> > +    latency = distance / unit
> Vaspol noted, but it's already in km right?
> 
> —
> Reply to this email directly or view it on GitHub.
>